code/modules/clothing/gloves/special.dm

```diff
@@ -49,6 +50,16 @@
 
 	QDEL_NULL(pull_component_weakref)
 
+/obj/item/clothing/gloves/cargo_gauntlet/security
```

Comment:
This might be more work than you're willing to do and I don't mind much, but I don't like this subtype at all. I think we should instead make a component/element for "adds components when worn, removes when taken off".